tools/testing-server/plugins/browserify/browserify-transform.js

```diff
@@ -31,6 +31,8 @@ function browserifyScript (scriptPath, enablePolyfills) {
           '@babel/plugin-syntax-dynamic-import',
           '@babel/plugin-transform-modules-commonjs',
           '@babel/plugin-proposal-optional-chaining',
+          '@babel/plugin-proposal-nullish-coalescing-operator',
+          '@babel/plugin-proposal-logical-assignment-operators',
```

Comment:
Should these same transforms be added to the webpack and npm build?

Reply:
https://caniuse.com/?search=null%20coalescing%20assignment I don't think they do; it just isn't supported under Node v15